File: Codigos en Java/Ejercicio1_Ape06.java
import java.util.ArrayList; 
public class Ejercicio1_Ape06 { 
public static void main(String[] args) { 
ArrayList<String> estudiantes = new ArrayList<>(); 
estudiantes.add("Ana García"); 
estudiantes.add("Luis Martínez"); 
estudiantes.add("María López"); 
estudiantes.add("Carlos Pérez"); 
estudiantes.add("Sofía Ramírez"); 
System.out.println("Lista de estudiantes:"); 
for (String nombre : estudiantes) { 
System.out.println("  - " + nombre); 
} 
System.out.println("Total: " + estudiantes.size()); 
} 
}

File: Codigos en C++/Ejercicio1_Ape06.cpp
#include <iostream> 
#include <vector> 
#include <string> 
using namespace std; 
int main() { 
vector<string> estudiantes; 
estudiantes.push_back("Ana García"); 
estudiantes.push_back("Luis Martínez"); 
estudiantes.push_back("María López"); 
estudiantes.push_back("Carlos Pérez"); 
estudiantes.push_back("Sofía Ramírez"); 
cout << "Lista de estudiantes:" << endl; 
for (const string& nombre : estudiantes) { 
cout << "  - " << nombre << endl; 
} 
cout << "Total: " << estudiantes.size() << endl; 
return 0; 
}